app/views/govuk_publishing_components/components/_phase_banner.html.erb

```diff
@@ -1,22 +1,27 @@
 <%
+app_name ||= nil
 phase ||= nil
+message ||= nil 
 
 unless phase.in?(%w[alpha beta])
   raise ArgumentError, "The phase banner component expects a `phase` (`beta` or `alpha`), #{phase.inspect} given"
 end
+
+unless message.present?
+  if phase == "beta"
+    message = raw("This part of GOV.UK is being rebuilt &ndash; <a href=\"/help/beta\">find out what beta means</a>")
+  elsif phase == "alpha"
+    message = raw("This part of GOV.UK is being built &ndash; <a href=\"/service-manual/phases/ideal-alphas\">find out what alpha means</a>")
+  end
+end 
 %>
 
-<div class="gem-c-phase-banner">
-  <p>
-    <strong class="phase-tag"><%= phase %></strong>
-    <span>
-      <% if local_assigns[:message] %>
-        <%= local_assigns[:message] %>
-      <% elsif phase == "beta" %>
-        This part of GOV.UK is being rebuilt &ndash; <a href="/help/beta">find out what beta means</a>
-      <% elsif phase == "alpha" %>
-        This part of GOV.UK is being built &ndash; <a href="/service-manual/phases/ideal-alphas">find out what alpha means</a>
-      <% end %>
-    </span>
-  </p>
-</div>
+<%= tag.div class: "gem-c-phase-banner" do %>
```

Comment:
👌 ace for using helpers for markup

Do we need two nested divs? I think we can have one with `class: "gem-c-phase-banner govuk-phase-banner"`